data/en/2020/hispano/agenda.yml

```diff
@@ -0,0 +1,102 @@
+complete: false
+types:
+    - name: Keynotes
+      id: keynote
+      color: "#e44"
+    - name: Panel
+      id: panel
+      color: "#c6a061"
+    - name: Talks
+      id: talk
+      color: "purple"
+    - name: Vendor Talks
+      id: vendor
+      color: "orange"
+
+items:
+    - name: 'Warm-up'
+      vod:
+      presenter: Committee
+      type: keynote
+      time: 9:00 - 9:30 (GMT-4)
+      abstract: <p>Warmup</p>
+    - name: 'Keynote'
+      vod:
+      presenter:
+      type: keynote
+      time: 9:30 - 10:00 (GMT-4)
+      abstract: <p>Keynote</p>
+    - name: Kubernetes Nativo con Eclipse MicroProfile
```

Comment:
Most of the names of the sessions are still in Spanish. Is that intended to go live?